application/routes.py

```diff
@@ -31,17 +31,34 @@ def zip_lists(a, b, c):
 # Register the custom filter with Jinja
 app.jinja_env.filters['zip'] = zip_lists
 
+# Define the custom Jinja filter
+@app.template_filter()
+def zip_lists_detection(a, b):
+    return zip(a, b)
+
+# Register the custom filter with Jinja
+app.jinja_env.filters['zip_detection'] = zip_lists_detection
+
 @app.route("/settings")
 def settings():
     status = []
-    recipients = vault.get_keys("recipients")
     numbers = []
+    cameras_status = []
+
+    recipients = vault.get_keys("recipients")
+    cameras = TFLite_detection_stream.CAMERAS.keys()
```

Comment:
We are now accessing a lot of things from this class. Maybe we should look into making the detection file a class? 🤔